istio-control/istio-autoinject/templates/mutatingwebhook.yaml

```diff
@@ -35,8 +35,6 @@ webhooks:
         operator: NotIn
         values:
         - disabled
-      - key: istio-env
```

Comment:
If you remove this I'll merge. But I don't think we want this, see discussion on https://github.com/istio/installer/pull/247